src/gui/previewselect.cpp

```diff
@@ -45,6 +46,11 @@ PreviewSelect::PreviewSelect(QWidget* parent, BitTorrent::TorrentHandle *const t
 {
     setupUi(this);
     setAttribute(Qt::WA_DeleteOnClose);
+
+    buttonBox->button(QDialogButtonBox::Ok)->setText(tr("Preview"));
+    connect(buttonBox->button(QDialogButtonBox::Ok), &QPushButton::clicked, this, &PreviewSelect::previewButtonClicked);
+    connect(buttonBox->button(QDialogButtonBox::Cancel), &QPushButton::clicked, this, &PreviewSelect::cancelButtonClicked);
```

Comment:
The same comment about these connections.

Reply:
should be fixed.